swarm/storage/database.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/syndtr/goleveldb/leveldb/opt"
 )
 
-const openFileLimit = 128
+const openFileLimit = 32
```

Comment:
Is this change from 128 to 32 safe for production?

Reply:
This is what I want to find out, but most probably it is.

var leveldbOptions = &opt.Options{
    Compression: opt.NoCompression,
    BlockSize:   1 << 16,
    // Default max open file descriptors (ulimit -n) is 256 on OS
    // X, and >=1024 on (most?) Linux machines. So set to a low
    // number since we have multiple leveldb instances.
    OpenFilesCacheCapacity: 10,
}
This are the params that `libkbfs` is using (https://github.com/keybase/kbfs).

We could run this branch on the staging cluster and monitor the number of open files there. Will do that and report back.

Reply:
Great. Thank you for clarification and the test. 👍